pkg/build/rpm.go

```diff
@@ -27,9 +36,23 @@ func (b *Builder) getRPMFileNames(rpmSourceDir string) ([]string, error) {
 	return rpmFileNames, nil
 }
 
-func (b *Builder) copyRPMs() error {
+func (b *Builder) copyRPMs(rpmSourceDir string, rpmDestDir string, rpmFileNames []string) error {
```

Comment:
Since you're passing in all of the information you need, as compared to having this method derive it from the state of Builder, this doesn't need to be bound to Builder.

Reply:
Yeah I was actually planning to ask about that, I agree with what you said